### ISSUE TITLE
Consider replace WebDriverManager by Selenium built-in SeleniumManager

### ISSUE THREAD

**Issue by rosolko** [rosolko/WebDriverManager.Net#260]
Hi,

I have to raise a point to discuss and verify - built-in SeleniumManager from Selenium project.

Is this tool able replace WebDriverManager with no drawback? 
If yes - from my point of view there is no more sence to support and evolve current app.

For example - starting from Selenium 4.11.0 built-in SeleniumManager also solve Chrome for Testing initiative.

https://www.selenium.dev/documentation/selenium_manager/

**Comment by ms6073** [comment]
I do not think you are going to get a definitive answer here so I suggest you take a look at the [Selenium Manager github](https://github.com/SeleniumHQ/selenium/tree/trunk/rust) page for more information.

**Comment by canarioPorCanarias** [comment]
just tried, looks a more easy way to get the chromedriver and cleaner, looks like it could replace it,
but this project has more configurations for very specific purposes but in general i think it will replace it

**Comment by inflextion** [comment]
I think it's a bit too early for that, Selenium Manager still needs some improvements.
Myself, I'm facing issues with proxies (to downloads chromedriver) while using Selenium Manager.
The solution to set proxy/ clear after download (as I don't need it for the browser itself) is not as clean.

**Comment by KeithDickson** [comment]
Selenium's built-in manager did indeed turn out to be a suitable replacement for me.

I just recently started using a web driver manager (after having manually managed my web drivers for many years). WebDriverManager.Net seemed a bit more attractive than Selenium Manager because the latter is still in "beta" status, and I was able to get WebDriverManager.Net working pretty quickly and easily. However, while checking on [issue #189](https://github.com/rosolko/WebDriverManager.Net/issues/189) which affected me, I came across the discussion on this issue and decided to take a closer look at the Selenium built-in offerings.

After giving it a try, it worked just as well for my use cases. I didn't even have to configure anything...it simply worked automatically. And I was pleased to find that it's already bundled with the Selenium NuGet package that I'm already importing (after, that is, I updated from the earlier version I was using from before Selenium Manager was even a thing).

So for me, the convenience and likely better long-term support of a built-in manager outweigh benefits of using this third-party web driver manager.

**Comment by jduust** [comment]
Using UiPath Studio i have a different view: The SeleniumManager doesn't work with UiPath due to pathing/environment errors, so i prefer this to the built in versions of selenium, but neither does the newer versions of WebDriverManager.Net due to a conflict with system.text.json (dll-hell) causing it to look for 7.0.0 instead of 7.0.3 which it's packaged with so it's kinda a lose-lose-situation right now due to bugs. I hope you keep this running until the seleniummanager is out of beta and runs without issues, this works way better for me at least, but i get why you would want to move on, it makes sense.

**Comment by rikrak** [comment]
Looks like the Selenium Manager requires a rust installation.  Personally I like this solution as it's quicker to get going, and does not require dev's to install and maintain otherwise unnecessary dependencies

**Comment by jduust** [comment]
> Looks like the Selenium Manager requires a rust installation. Personally I like this solution as it's quicker to get going, and does not require dev's to install and maintain otherwise unnecessary dependencies

It doesn't require rust, and it's included in newer versions of selenium by default. It works well with standard development environments from my experience, but requires some tweaking if you use less popular solutions.
